ethsigner/app/src/main/java/tech.pegasys.ethsigner/ApplicationInfo.java

```diff
@@ -13,7 +13,7 @@
 package tech.pegasys.ethsigner;
 
 public class ApplicationInfo {
-  private static final String CLIENT_IDENTITY = "ethsigner" + "";
+  private static final String CLIENT_IDENTITY = "tech/pegasys/ethsigner" + "";
```

Comment:
What's the deal with adding an empty string?

Reply:
refactor ... bugger.